_includes/context/priority.md

```diff
@@ -1,3 +1,98 @@
+
+### Troubleshooting context toolbar and context form conflicts
+
+There are situations where custom context toolbars or custom context forms may conflict with:
+
+  - Context toolbars or context forms provided by the editor,
+  - Context toolbars or context forms provided by the [Quick Toolbars (`quickbars`) plugin]({{site.baseurl}}/plugins/quickbars/),
+  - Other custom context toolbars or custom context forms.
+
+{% if site.productmajorversion < 6 %}
+How these conflicts are resolved depends on the {{site.productname}} version.
+
+- For {{site.productname}} 5.3 or newer, see: [Determining the display priority of context toolbars and context forms]().
+- For {{site.productname}} 5.0, 5.1, or 5.2; see: [Determining the display priority of context toolbars and context forms (Legacy behavior)]().
+{% endif %}
+
+#### Determining the display priority of context toolbars and context forms
+
+There are three settings that determine the priority of context toolbars and context forms: `scope`, `predicate`, and `position`.
+
+- `scope` - Sets the context toolbar or form as either: specific to certain types of content (`node`), or a general (global) toolbar or form (`editor`).
+- `predicate` - A function that tests in the current context position against one or more conditions, returning a boolean value.
+- `position` - Sets where the context toolbar or form is rendered, relative to the current section (`selection`, `node`, or `line`).
+
+Generally:
+
+- Context _forms_ are prioritized over context _toolbars_.
+- Context _forms_ with `scope: 'node'` are prioritized over `scope: 'editor'`.
+- Only one context _form_ can be shown for a selection or cursor position, they cannot be concatenated to another context _form_ or a context _toolbar_.
+- Context _toolbars_ with `position: 'selection'` are prioritized over `position: 'node'`, and `position: 'line'` is given the lowest priority.
+- The editor concatenates context _toolbars_ when there is more than one context _toolbar_ to display with:
+
+    - A matching predicate,
+    - The same `scope` and `position` values.
+
+  Concatenated toolbars may contain duplicate toolbar items.
+
+- If no matching context toolbars or context forms are found for the selection or cursor position, then editor will recursively search for matches on the parent node of the current node, until it reaches the root node of the editor content.
+
+##### Description of how context toolbars and context forms are prioritized
+
+The following description can be used for troubleshooting the behavior of context toolbars and context forms.
+
+The editor will determine which context toolbars or context form will be shown using following process:
+
+1. Find all **context forms** with both:
+
+    - `scope: 'node'`
+    - A predicate matching the current selection or cursor position.
+
+    If there any matching **context forms**, the first one found will be displayed in the editor and the process will end.
```

Comment:
Should this be 'If there are any...'? Same sentence structure is present multiple times below as well.

Reply:
:facepalm: thanks

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1517/commits/957488d197547b25cda061bfce54563b2a350c73